File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C/session1/p7.c
#include<stdio.h>
void main(){

	int num=9;
	
	for(int i=3;i>=1;i--){

		for(int j=3;j>=1;j--){

			printf("%d",num);
			num--;
		}
		printf("\n");
	}
}

File: java-main/java-main/IO/Nestedfor6/p7.java
/*Write a program to take range as input from the user and print the reverse of all numbers.*/

import java.io.*;

class Demo{
	public static void main(String[] args)throws IOException{
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter Start:");
		int startRange=Integer.parseInt(br.readLine());
		System.out.println("Enter End:");
		int endRange=Integer.parseInt(br.readLine());

		System.out.println();
		for(int i=startRange;i<=endRange;i++){
			int num=i;
			int rev=0;
			while(num!=0){
				int rem=num%10;
				rev=rev*10+rem;
				num=num/10;
			}
			System.out.println(rev);
		}
	}
}
